test/icalendar/deserialize_test.exs

```diff
@@ -36,6 +36,30 @@ defmodule ICalendar.DeserializeTest do
       assert event.dtend.time_zone == "America/Chicago"
     end
 
+    test "with RRULE" do
+      rrule = [
+        "FREQ=DAILY",
+        "UNTIL=22221224T084500Z",
+        "BYMONTHDAY=1,3,5",
+        "BYDAY=TU,FR",
+        "BYMONTH=4"
+      ] |> Enum.join(";")
+
+      ics = """
+      BEGIN:VEVENT
+      RRULE:#{rrule}
+      END:VEVENT
+      """
+
+      event = ICalendar.from_ics(ics)
```

Comment:
Now that I think about it I think this should probably return an `ok` tuple as parsing can fail. What do you think?